src/did/Did.ts

```diff
@@ -65,7 +65,7 @@ export interface IDidDocumentPublicKey {
   publicKeyHex: string
 }
 
-export interface IDidDocumentPpties {
+export interface IDidDocumentProperties {
```

Comment:
we export this, so I wouldn't want to change the name

Reply:
dependencies can be fixed quick, therefore I think it is worth the compatibility patching and having the name be in line with our naming convention.

Reply:
In our github we only have one consumer of this interface

Reply:
fine with it if you attach a PR that fixes the consumer

Reply:
This PR fixes the only consumer.